bindings/nodejs/examples/package.json

```diff
@@ -4,7 +4,8 @@
     "main": "dist/index.js",
     "private": true,
     "scripts": {
-        "build": "tsc"
+        "build": "tsc",
+        "prebuild": "npm install"
```

Comment:
Why make this an own command?

Reply:
my bad, was just annoyed i had to build sdk each time. probably missed something about node haha